app-i18n/mozc/mozc-2.23.2785.102.recipe

```diff
@@ -2,17 +2,18 @@ SUMMARY="Japanese input method editor"
 DESCRIPTION="A Japanese Input Method Editor (IME) designed for multi-platform. \
 This is open source version."
 HOMEPAGE="https://github.com/google/mozc/"
-COPYRIGHT="2010-2016, Google Inc."
+COPYRIGHT="2010-2018, Google Inc."
 LICENSE="BSD (3-clause)"
-REVISION="2"
-fileVersion="v2.20.2677.102.6"
+REVISION="1"
+fileVersion="v2.23.2785.102.1"
 SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
-CHECKSUM_SHA256="cdb43a88d82db185299a8e396fccfe76ddbbe5a3c6b219220a36b55115cff4ac"
+CHECKSUM_SHA256="17ce4ee5d6a55678d804890d7e2fa7ffd7f852a9673e3b79671613c9167469f5"
 SOURCE_DIR="mozc"
-if [ $effectiveTargetArchitecture == "x86" ]; then
+if [ $effectiveTargetArchitecture = x86 ]; then
 	PATCHES="mozc-$portVersion-x86.patch"
 fi
-SOURCE_URI_2="https://jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/0.6.0-20080911/zinnia-tomoe-0.6.0-20080911.tar.bz2"
+zinniaTomoeVersion="0.6.0-20080911"
+SOURCE_URI_2="https://jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/$zinniaTomoeVersion/zinnia-tomoe-$zinniaTomoeVersion.tar.bz2"
```

Comment:
```diff
-SOURCE_URI_2="https://jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/$zinniaTomoeVersion/zinnia-tomoe-$zinniaTomoeVersion.tar.bz2"
+SOURCE_URI_2="https://downloads.sourceforge.net/zinnia/zinnia-tomoe-$zinniaTomoeVersion.tar.bz2"
```
(You forgot to replace `jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/` by `https://downloads.sourceforge.net/zinnia/`.)

BTW, what we are doing here is that we make use of SourceForge's redirection service. This has 2 advantages: We let SF choose the best mirror, and we also let SF find which subdirectory the archive is in. This means this URI will still be valid if upstream ever decides to change the directory hierarchy.